backend/codesirius/problems/models/sample_test.py

```
@@ -0,0 +1,14 @@
+from django.db import models
+
+from problems.models import Problem
+
+
+class SampleTest(models.Model):
+    problem = models.ForeignKey(
+        Problem, on_delete=models.CASCADE, related_name="sample_tests"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)